documentation/markup.rst

```diff
@@ -42,7 +66,9 @@ The standard reST inline markup is quite simple: use
 
 * one asterisk: ``*text*`` for emphasis (italics),
 * two asterisks: ``**text**`` for strong emphasis (boldface), and
-* backquotes: ````text```` for code samples.
+* backquotes: ````text```` for code samples, variables, and literals.
+
+Italic is also used for function and method arguments (``*arg*``).
```

Comment:
The one thing to think about here is that this section currently is nearly word for word identical to the corresponding one in the [Sphinx primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup), and these would be the only CPython-specific additions. Depending on what we decide to do about that, this could be either desirable or not, as on one hand only including it below would allow us to elide the section completely or at least just make it a link to the Sphinx version and keep everything in one place, but on the other it would give us _something_ to say here if we keep the section.

Also, to note, the mention in the roles section is not consistent with here, and directly conflicts with what is stated in the table, as it says:

> Names of local variables, such as function/method arguments, are an exception,
they should be marked simply with ``*var*``.

When the table above says that variables should be formatted as inline literals:

> variables ->  ``` ``var`` ```

The second (as implemented here) seems simpler, more sensible and more consistent with what is actually done in CPython's docs, so the other one should be modified accordingly (assuming we keep it around).